test/module/irohad/model/command_validate_execute_test.cpp

```diff
@@ -1061,3 +1061,42 @@ TEST_F(RevokePermissionTest, InvalidCaseNoPermissions) {
       .WillOnce(Return(false));
   ASSERT_FALSE(validateAndExecute());
 }
+
+class SetAccountDetailTest : public CommandValidateExecuteTest {
+ public:
+  void SetUp() override {
+    CommandValidateExecuteTest::SetUp();
+
+    pubkey_t creator_key, account_key;
```

Comment:
why keys are required? they seem unused

Reply:
They are used in some tests

Reply:
You mean will be used? Didn't find the usage in the current code